packages/editor/src/operations/AppModelManager.ts

```diff
@@ -22,6 +24,16 @@ function genSlotTrait(parentId: string, slot: string): ComponentTrait {
     },
   };
 }
+// TODO JSONSchema7Type
+const genInitProps = (init: Record<string, never>, properties: any, fullKey?: string) => {
```

Comment:
there is a similar tool function `parseTypeBox` in runtime. Maybe we can reuse it.

Reply:
Thanks, I‘m already used this function and moved it to before rendering component form